prepare_check_cleanup/reconcile.py

```diff
@@ -71,6 +82,15 @@ def main() -> None:
                     urls_with_actions=urls_with_actions, discourse=discourse, **action_kwargs
                 )
             )
+        case Action.PREPARE_UPDATE.value:
+            prepare_update(**action_kwargs)
```

Comment:
`action_kwargs` is deliberately general in nature since it is intended to provide arguments that are unique to each of the possible actions that can be taken. Type hints are complex and not that useful in those kind of cases, especially since this is mainly used during testing